src/mono/nuget/Microsoft.NET.Sdk.WebAssembly.Pack/build/Microsoft.NET.Sdk.WebAssembly.Browser.targets

```diff
@@ -333,13 +336,26 @@ Copyright (c) .NET Foundation. All rights reserved.
       <Output TaskParameter="AssetsWithTargetPath" ItemName="_WasmJsModuleCandidatesForBuildWithTargetPath" />
     </ComputeStaticWebAssetsTargetPaths>
 
-    <GetFileHash Files="@(_WasmJsModuleCandidatesForBuildWithTargetPath)" Algorithm="SHA256" HashEncoding="base64">
-      <Output TaskParameter="Items" ItemName="_WasmOutputWithHash" />
-    </GetFileHash>
+    <DefineStaticWebAssetEndpoints
+      CandidateAssets="@(WasmStaticWebAsset)"
+      ExistingEndpoints="@(StaticWebAssetEndpoint)"
+      ContentTypeMappings="@(StaticWebAssetContentTypeMapping)"
+    >
+      <Output TaskParameter="Endpoints" ItemName="WasmStaticWebAssetEndpoint" />
+    </DefineStaticWebAssetEndpoints>
+
+    <ResolveFingerprintedStaticWebAssetEndpointsForAssets
+      CandidateEndpoints="@(WasmStaticWebAssetEndpoint)"
+      CandidateAssets="@(WasmStaticWebAsset)"
```

Comment:
Passing here also `_WasmJsModuleCandidatesForBuildEndpoint` and `_WasmJsModuleCandidatesForBuild` removes the need to use the target path.